common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py

```diff
@@ -222,14 +199,13 @@ def _create_a_timed_exam_and_attempt(self):
         # Start the proctored exam.
         self.courseware_page.start_timed_exam()
 
-    @flaky  # TODO fix this, see SOL-1183
+    @flaky(max_runs=10, min_passes=10)
     def test_can_add_remove_allowance(self):
         """
         Make sure that allowances can be added and removed.
         """
-
-        # Given that an exam has been configured to be a proctored exam.
-        self._create_a_proctored_exam_and_attempt()
+        # Given that an exam has been configured to be a timed exam.
+        self._create_a_timed_exam_and_attempt()
```

Comment:
All the other test use `create_a_timed_exam_and_attempt` and they are passing without any flakiness.